docs/paper/admin/how-to/anti-xray.md

```diff
@@ -135,11 +135,15 @@ Copy and paste into your `paper-world.yml` within your nether world folder. See
 ```yml title="world_nether/paper-world.yml"
 anticheat:
   anti-xray:
+    enabled: true
+    engine-mode: 1
     hidden-blocks:
     - ancient_debris
     - nether_gold_ore
     - nether_quartz_ore
     max-block-height: 128
```

Comment:
```suggestion
    lava-obscures: false
    max-block-height: 128
```

Same for engine-mode: 2 below. Sorry, I can't do it via suggestion because it's not part of the diff.